src/main/resources/META-INF/plugin.xml

```diff
@@ -340,6 +340,11 @@
                          enabledByDefault="false" level="WARNING"
                          implementationClass="org.rust.ide.inspections.RsSimplifyBooleanExpressionInspection"/>
 
+        <localInspection language="Rust" groupName="Rust"
+                         displayName="[Experimental] No mutable required"
```

Comment:
Let's don't put "[Experimental]" here. I think the caveat in the description is enough.